.github/workflows/refresh.yml

```diff
@@ -0,0 +1,18 @@
+name: Refresh
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 3 * * *' # Runs every day at 3am
```

Comment:
Why is it necessary to refresh the pages every day?

Reply:
probably not strictly necessary, just in case there is a build issue and we dont realize it to manually trigger it...